doc/source/whatsnew/v2.1.0.rst

```diff
@@ -149,7 +149,7 @@ Other API changes
 
 Deprecations
 ~~~~~~~~~~~~
-- Deprecated :meth:`.DataFrameGroupBy.apply` operating on the grouping column(s) (:issue:`7155`)
+- Deprecated :meth:`.DataFrameGroupBy.apply` and :class:`.Resampler` methods operating on the grouping column(s); subset the data to exclude the grouping column(s) to adopt future behavior and avoid the ``FutureWarning`` (:issue:`7155`)
```

Comment:
This PR has no affect `Resampler` itself, right?

I'm not sure I would understand the phrase "and :class:`.Resampler` methods operating on the grouping column(s)". Instead maybe "and the methods in :meth:`DataFrameGroupBy.resample`  operating on the grouping column(s).

Reply:
> This PR has no affect `Resampler` itself, right?

True, it's technically DatetimeIndexResamplerGroupby, PeriodIndexResamplerGroupby, and TimedeltaIndexResamplerGroupby. 

> the methods in :meth:DataFrameGroupBy.resample

I think "the methods on the objects returned by :meth:DataFrameGroupBy.resample...